choice/basicterm.py

```diff
@@ -70,7 +77,7 @@ def ask(self):
                     break
             else:
                 # Global action?
-                for ga in self.global_actions:
+                for ga in self.global_actions or []:
```

Comment:
Nice catch! I think you could put this at the top of the class:

```
class BasicTermMenu(object):
  def __init__(...):
    self.global_actions = idNameList(global_actions) or []
```

Or alternatively, edit `idNameList()` to return `[]` when the input is None. This would prevent other crashes, e.g. here:

https://github.com/fyhuang/choice/pull/3/files#diff-ea6df2eb5ce078088b03f5d6e6365c16L33